op-enable.go

```diff
@@ -164,7 +149,7 @@ func enable(he vmextension.HandlerEnvironment, d driver.DistroDriver) error {
 
 // installCompose download docker-compose and saves to the specified path if it
 // is not already installed.
-func installCompose(path string) error {
+func installCompose(path string, composeUrl string) error {
```

Comment:
composeUrl used to be a global variable (therefore naming `composeUrl` used to make sense). now it should be just `url`.